packages/@lwc/engine/scripts/rollup/rollup.config.js

```diff
@@ -12,6 +12,9 @@ const { version } = require('../../package.json');
 const entry = path.resolve(__dirname, '../../src/framework/main.ts');
 const targetDirectory = path.resolve(__dirname, '../../dist/');
 
+const rollupLwcFeatures = require('@lwc/features/src/rollup-plugin');
```

Comment:
I'm not so sure about how these plugins are accessed but I do think that they belong in the same package.

Reply:
A package should either expose a babel transformer or a rollup plugin. I don't think this package should expose both.

IMO, I would rather export a babel plugin instead and not a rollup config since a can use the babel transform via `babel-rollup-plugin`.

Reply:
@pmdartus `@lwc/engine` currently uses rollup so unless we want to refactor the current build, we should probably go with the rollup plugin. Also, it seems that `babel-rollup-plugin` is designed to generate a rollup bundle from files compiled by babel, which does not fit the use case for `@lwc/engine`.

Let me know if I misunderstood your feedback. If you prefer, I can move the rollup plugin wrapper into `@lwc/engine` since it's basically a one-liner.